pom.xml

```diff
@@ -137,6 +137,24 @@
       <scope>provided</scope>
       <version>3.6.0</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.11.0</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <version>1.2.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.11</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I do not think that this is the way to go here...

Maven do not use `logback` or `log4j`, it uses it's own apache logger. You can access it in the `AbstractMojo#getLog()` method. We need to use this logger for any logging.